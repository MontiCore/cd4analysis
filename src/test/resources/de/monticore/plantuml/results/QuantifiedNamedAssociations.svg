<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentScriptType="application/ecmascript" contentStyleType="text/css" height="279px" preserveAspectRatio="none" style="width:265px;height:279px;background:#000000;" version="1.1" viewBox="0 0 265 279" width="265px" zoomAndPan="magnify"><defs/><g><rect fill="#DDDDDD" height="27.6094" rx="5" ry="5" style="stroke: #000000; stroke-width: 1.0;" width="228" x="31" y="8"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacingAndGlyphs" textLength="216" x="37" y="27.5332">.QuantifiedNamedAssociations CD</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="140" x="5" y="63.6094">Dot Executable: null</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="173" x="5" y="81.2188">No dot executable found</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="258" x="5" y="98.8281">Cannot find Graphviz. You should try</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="0" x="9" y="116.4375"/><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="71" x="5" y="134.0469">@startuml</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="50" x="5" y="151.6563">testdot</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="66" x="5" y="169.2656">@enduml</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="0" x="9" y="186.875"/><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="15" x="9" y="204.4844">or</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="0" x="9" y="222.0938"/><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="201" x="5" y="239.7031">java -jar plantuml.jar -testdot</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="0" x="9" y="257.3125"/><text fill="#888888" font-family="sans-serif" font-size="10" lengthAdjust="spacingAndGlyphs" textLength="200" x="34" y="276.3027">generated with MontiCore using PlantUML</text><!--MD5=[16ff5bfb170e0960774ee7a5f58b3356]
@startuml
<style>
	classDiagram {
		class {
			BackgroundColor White
			RoundCorner 0
	  }
	  legend {
      BackgroundColor White
      RoundCorner 0
    }
</style>
hide circle
hide empty members

  skinparam classAttributeIconSize 0
legend top right
  .QuantifiedNamedAssociations CD
end legend
class One {
  {field} String name
  {field} int number
  {field} boolean exists
}

class Two {
  {field} String name
}

class Three {
  {field} String name
}

class Four {
  {field} String name
}

One "b 1" - -* "1" Two : a

Two "d 1..*" *- -* "e 1" Three : c

Three "g 1" *- - "h *" Four : f

Four "i *" - -* "j 1" One

Two "1..*" - -* "k 1..*" One : l

center footer generated with MontiCore using PlantUML
@enduml

PlantUML version 1.2020.15(Sun Jun 28 13:39:45 CEST 2020)
(GPL source distribution)
Java Runtime: IBM Semeru Runtime Open Edition
JVM: Eclipse OpenJ9 VM
Default Encoding: windows-1252
Language: de
Country: DE
--></g></svg>
