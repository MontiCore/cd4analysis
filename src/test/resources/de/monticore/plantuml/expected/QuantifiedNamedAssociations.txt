@startuml
  skinparam classAttributeIconSize 0
  skinparam legend {
    BorderColor black
    BackGroundColor white
  }
  legend top right
    .QuantifiedNamedAssociations CD
  end legend
  class One {
    {field} String name
    {field} int number
    {field} boolean exists
  }

  class Two {
    {field} String name
  }

  class Three {
    {field} String name
  }

  class Four {
    {field} String name
  }






  Two "d" *--* "e" Three
  Two "i" --* "j" One
  Four "g" --* "h" One : c
  center footer generated with MontiCore using PlantUML
@enduml
